sources/packages/backend/libs/integrations/src/services/msfaa-number/msfaa-number.service.ts

```diff
@@ -54,6 +54,7 @@ export class MSFAANumberService extends RecordDataModelService<MSFAANumber> {
       .innerJoin("currentAssessment.offering", "offering")
       .innerJoin("offering.institutionLocation", "institutionLocation")
       .where("msfaaNumber.dateRequested is null")
+      .andWhere("msfaaNumber.cancelledDate is null")
```

Comment:
Curious, the generation of pending msfaa for a cancelled record, should have a dateRequested already right? or are we considering the old data that was imported for MSFAA?

Reply:
Mine though was to add this condition in preparation for the upcoming requirement for MSFAA reactivation which would potentially have SIMS setting the canceled date.
From #1777 
_Update ANY OTHER active (with null cancellation) MSFAA numbers for the student for the SAME OFFERING INTENSITY by populating the cancelled date and a boolean column indicating it is a system cancellation._

Either way, sending a canceled MSFAA would be a mistake.

Reply:
@guru-aot with the upcoming requirement of re-activating MSFAA number, A cancelled MSFAA number can be re-activated where a pending msfaa(date_signed is null) could be cancelled by SIMS. At this situation, for that msfaa record which is cancelled will never have date signed.